lib/controllers/users.js

```diff
@@ -0,0 +1,44 @@
+const { Router } = require('express');
+const User = require('../models/User');
+const UserService = require('../services/UserService');
+const authenticate = require('../middleware/authenticate');
+const authorize = require('../middleware/authorize');
+
+const ONE_DAY_IN_MS = 1000 * 60 * 60 * 24; 
+
+module.exports = Router()
+  .post('/', async (req, res, next) => {
+    try {
+      const [, user] = await UserService.signUp(req.body);
+      res.json(user);
```

Comment:
For this assignment, we were looking for the user to be signed up and also signed in all within this route.